test_simple_pe/test_simple_pe_magma.py

```diff
@@ -59,7 +67,141 @@ def test_simple_pe():
 
 
 """  # noqa
-    with open("test_simple_pe/build/problem.txt", "w") as f:
+    with open("test_simple_pe/build/problem_pe_core.txt", "w") as f:
         f.write(problem)
     assert not os.system(
-        "CoSA --problem test_simple_pe/build/problem.txt")
+        "CoSA --problem test_simple_pe/build/problem_pe_core.txt")
+
+
+@pytest.mark.parametrize("ops", ops)
+def test_simple_pe(ops):
+    pe = define_pe(ops, T=m.UInt, data_width=16)
+
+    class SimplePETester(ResetTester, ConfigurationTester):
+        pass
+
+    pe_functional_model = gen_simple_pe(ops, 16)()
+
+    tester = SimplePETester(pe, pe.clk, pe_functional_model)
+
+    m.compile("test_simple_pe/build/pe", pe, output="coreir",
+              coreir_args={"passes": ["rungenerators", "flatten",
```

Comment:
we need to change this as well right? ala #96

Reply:
also maybe the cause for the error below

Reply:
done